taskcluster/app_services_taskgraph/transforms/module_build.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.transforms.base import TransformSequence
+
+from taskcluster.app_services_taskgraph.build_config import script_to_bash_command
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def release_upload_symbols(config, tasks):
+    for task in tasks:
+        if config.params["tasks_for"] != "github-release":
+            yield task
+            return
```

Comment:
I only want to upload symbols if this is a github release. I implemented it as a guard. The `yield` doesn't end the function, so if I didn't `return`, then we'd always upload symbols.
I'll refactor this to not use a guard

Reply:
Oh, I realize now that that `return` could've caused issues with `yield`-ing, since only one `yield` would've happened.
The new implementation I've done since your comment shouldn't have that issue